### ISSUE TITLE
public_html structure - use of subdirectory

### ISSUE THREAD

**Issue by phpitech** [phpList/phplist3#394]
Related (but different) discussion @ #336 

The current structure for public_html assumes a /lists folder on a (sub)domain, and requires a user to manually edit the index.html file to reflect their own website when installing phplist.

While I can see why that may have been done historically, in today's internet where the use of reverse proxies / rewrite rules etc is common, and the concept of not putting sites at common locations (e.g. /lists) to avoid automated vulnerability scanners, it would be nice to see 'public_html' contain the source of the application itself. 

The current structure looks to be a historical decision ?

Having just been going through the installation process and thinking of ease of use for a new user, this caught me out initially; I first configured the webroot for /, then for public_html then realised that I needed to either use /public_html/lists or edit the index file - so between my mistake and this, made what should have been a simple 1 step into 3 steps

**Comment by samtuke** [comment]
@phpitech Good points. 
@michield Thoughts?

**Comment by michield** [comment]
Yes, it's fairly historical, but I do think the installation instructions are correct. So, you're mostly saying, we should bridge the gap between your expectations and the reality, as most people won't read installation instructions but simple do what they assume needs to be done (which has changed over time). 

I think with things like docker on the way, all this will be less relevant. The current method works for people with a simple site who need a newsletter added as a function. Anyone going more "pro" than that will be able to figure it out. Also, I did it this way to avoid people saying "hey, installing phpList wiped my website", which, I kid you not, has happened.

**Comment by joshmaines** [comment]
It's easy to figure out on your own if you know how to code, as @michield said, but perhaps we should consider making these changes for people who don't really know anything about PHPList or PHP for that matter? Granted, it's a low priority task for sure.
